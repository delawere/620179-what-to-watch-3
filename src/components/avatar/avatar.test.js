import React from "react";
import renderer from "react-test-renderer";
import Avatar from "./avatar.jsx";

const props = {
  isAuth: true,
  avatarUrl: `testURL`
};

it(`Avatar renders correctly`, () => {
  const tree = renderer
    .create(
        <Avatar {...props} />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
