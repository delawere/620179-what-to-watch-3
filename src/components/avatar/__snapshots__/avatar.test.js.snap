// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar renders correctly 1`] = `
<div
  className="user-block__avatar"
>
  <img
    alt="User avatar"
    height="63"
    src="https://htmlacademy-react-3.appspot.comtestURL"
    width="63"
  />
</div>
`;
